apprating/src/main/java/com/mindera/skeletoid/apprating/AppRatingInitializer.kt

```diff
@@ -0,0 +1,56 @@
+package com.mindera.skeletoid.apprating
+
+import android.content.Context
+import com.mindera.skeletoid.apprating.callbacks.AppRatingDialogCallback
+import com.mindera.skeletoid.apprating.callbacks.AppRatingDialogResponse
+import com.mindera.skeletoid.apprating.callbacks.AppRatingDialogResponseCallback
+import com.mindera.skeletoid.apprating.controller.AppRatingController
+
+/**
+ * Entry point to setup and call the app rating dialog
+ */
+object AppRatingInitializer {
+
+    private val controller: AppRatingController by lazy {
+        AppRatingController()
+    }
+
+    /**
+     * Sets up the conditions to prompt the dialog.
+     * If it's not called, promptDialog will prompt the rating dialog immediately.
+     *
+     * @param countsPerTimeInterval Pair<Int, Long> values with the maximum number of times the dialog can be prompt per time range (in days)
+     * @param promptTimeInterval Time distance between prompts (in days)
+     */
+    fun init(countsPerTimeInterval: Pair<Int, Long>, promptTimeInterval: Long) {
```

Comment:
Do you see value in having default parameters?

Reply:
I guess we can have default parameters. That way if people using the lib were fine with that, they wouldn't have to set that up. What values do you think would suit as default?

Reply:
I'd say what we have on the app could be a good default